src/sqlfluff/core/rules/std/L043.py

```diff
@@ -153,9 +153,20 @@ class Rule_L043(BaseCrawler):
 
     | **Anti-pattern**
     | Querying all columns using `*` produces a query result where the number
-    | or ordering of columns may vary due to schema changes in upstream data
-    | sources. This should be avoided because it is prone to breakage in
-    | production.
+    | or ordering of columns changes if the upstream table's schema changes.
+    | This should generally be avoided because it can cause slow performance,
+    | cause important schema changes to go undetected, or break production code.
+    | For example:
+    | * If a query is expected to return a particular set of columns `c`, they
```

Comment:
Is this meant to say `c.*`?

Reply:
Good catch! I think I meant to list several column names. I'll fix this up.